plugins/modules/xcc_redfish_command.py

```diff
@@ -556,7 +556,7 @@ def raw_patch_resource(self, resource_uri, request_body):
         data = response["data"]
         for key in request_body.keys():
             if key not in data:
-                return {"ret": False, "msg": f"Key {key} not found. Supported key list: {data.keys()}"}
+                return {"ret": False, "msg": f"Key {key} not found. Supported key list: {list(data.keys())}"}
```

Comment:
While I think the above is valid for `CATEGORY_COMMANDS_ALL`, it's not really valid for `data`, since one line above the code says `if key not in data:`. Then why should it suddenly use `data.keys()` when it didn't use that here?